libraries/AP_Follow/AP_Follow.cpp

```diff
@@ -212,6 +216,12 @@ const AP_Param::GroupInfo AP_Follow::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("_JERK_H", 17, AP_Follow, _jerk_max_h_degsss, 360.0),
 
+    // @Param: _TIMEOUT
+    // @DisplayName: Follow timeout
+    // @Description: Follow position update from lead - timeout after x milliseconds
+    // @User: Standard
+    // @Units: ms
+    AP_GROUPINFO("_TIMEOUT", 18, AP_Follow, _timeout, AP_FOLLOW_TIMEOUT_DEFAULT),
```

Comment:
Got to a float and use seconds? Ah, looks like its a float already, just need to change the units.

Reply:
In fact I think its being used as seconds down on 966, so just the metadata needs updating.

Reply:
Yes - done.